api/pom.xml

```diff
@@ -147,25 +166,36 @@
                         <artifactId>maven-compiler-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>jdk-9</id>
+                                <id>jdk-11</id>
                                 <goals>
                                     <goal>compile</goal>
                                 </goals>
                                 <configuration>
-                                    <source>9</source>
-                                    <target>9</target>
+                                    <source>11</source>
+                                    <target>11</target>
```

Comment:
keep 9 here, it should be allowed to run on jdk9 properly

Reply:
Sure, if java9 profile is removed it makes sense keeping 9 here.